tests/unit/cli/admin/test_ceremony.py

```diff
@@ -83,6 +104,11 @@ def test_ceremony__non_positive_expiration(
             "",  # Please enter days until expiry for root role (365)
         ]
 
```

Comment:
Will you add the comment or not:
```suggestion
        # public keys and signing keys selection options
```